.devcontainer/Dockerfile

```
@@ -8,7 +8,7 @@ ARG VARIANT="bullseye"
 FROM mcr.microsoft.com/vscode/devcontainers/rust:0-${VARIANT}
 
 RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
-    && apt-get -y install --no-install-recommends git-lfs cmake protobuf-compiler libprotobuf-dev
+    && apt-get -y install --no-install-recommends git-lfs cmake protobuf-compiler libprotobuf-dev libssl-dev build-essential cmake clang
```

`cmake` is mentioned twice:

```suggestion
    && apt-get -y install --no-install-recommends git-lfs cmake protobuf-compiler libprotobuf-dev libssl-dev build-essential clang
```

nit: Someone will forget, so should we comment here why these new dependencies are needed? This way, they can remove them if ever Maho MQTT is no longer being used.